test/utils/scaffold.spec.ts

```diff
@@ -52,5 +52,5 @@ describe('Scaffold Utils', () => {
     await buildScaffold(projectDir, false);
 
     expect(fs.readdirSync(projectDir).sort()).toMatchSnapshot();
-  }, 60000);
+  }, 120000);
```

Comment:
Please check if this increase is really necessary (maybe a smaller value is also fine for your machine).